### DISCUSSION TITLE
Contact form doesn't work on website

### DISCUSSION THREAD

**Discussion by ER-95** [MobiusSync/MobiusSync#164] [Q&A] [answered]
Hi, I'm trying to send an email via the contact form on the website as I'm unable to restore my Pro access on my new iPad, but after clicking submit, it just spins infinitely - I can't find another way of contacting you so hoping I can get in touch here!
![image](https://github.com/MobiusSync/MobiusSync/assets/53078703/173f94bc-3e0f-43c3-b6c5-67aa4ebe28d0)

**Comment by SimonPickup** [comment]
Sorry for the trouble in making contact. You can also DM on twitter.

What exact problems are you seeing? Are you trying to Restore Purchase on the regular app or having trouble with the Pro app?

**Comment by ER-95** [comment]
@SimonPickup I don't use Twitter I'm afraid, apologies! Do you happen to have an email address I could use? I'd rather not share my Apple ID etc publicly if possible.

I'm trying to restore it on the regular app - it seems to try, then says "unable to restore purchase".

**Reply by SimonPickup** [reply]
I understand. We're in the process of setting up a new support email as Twitter is becoming more difficult for everyone.

You can try [Apple's suggestions here](https://support.apple.com/en-us/108096), and if all else fails you may have to ask Apple for a refund.

In any case, there's nothing we can do to re-enable your access as it is all handled through Apple with your Apple ID and we simply ask Apple's servers if you have the right to the purchase. We don't ever see your Apple ID.

I can see there's a bit more error detail we can surface in the error message, but that won't help you in the short term.

Out of interest what happens if you click "Purchase Now..."? Does it display a price and ask for Apple Pay approval or give a different error?

**Reply by ER-95** [reply]
I tried everything listed there prior to making contact to no avail, and upon speaking to Apple last night, they weren't able to refund something that was purchased so far in the past.

If I click "Purchase Now", it prompts me to purchase it again.

Is there any way to download the Pro version of the app to see if that would reinstate the access? On the App Store, it still shows as £4.99, but I'm not sure if there's a way around this.

**Reply by ER-95** [reply]
@SimonPickup Sorry to be a pain - not sure if my reply automatically notified you or not so though it'd be best to tag you just in case!

**Reply by SimonPickup** [reply]
The only thing I can suggest is you can use the beta of Mobius Sync Pro currently available on TestFlight, but it won't be available that way forever. https://github.com/MobiusSync/MobiusSync/discussions/156

**Reply by ER-95** [reply]
@SimonPickup Thank you for the info - is there any way of getting a more permanent solution? I'm really not sure what to do, and at the moment I'm not really in a position to be repurchasing something I already own!

**Reply by SimonPickup** [reply, marked as answer]
I don't think there's anything we can do, but if you want to keep discussing, we now have a trial support system set up. Please email here: trysupport@pickupinfinity.zohodesk.com
